src/branch_view.js

```diff
@@ -23,7 +46,7 @@ function renderHtmlBlob(parent, html) {
 	parent.insertBefore(banner, parent.firstChild);
 
 	banner_utils.addClass(banner, 'branch-banner-is-active');
-	banner.marginTop = '76px';
+	// banner.marginTop = '76px';
```

Comment:
I was going back-and-forth on it, because I didn't remember why it was there, and left this as a note.  But you're right, I'll remove it.